src/engine/sync/abletonlink.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include <ableton/Link.hpp>
+#include <ableton/link/HostTimeFilter.hpp>
+#include <ableton/platforms/stl/Clock.hpp>
```

Comment:
`ableton::Link::Clock` should also be included in `Link.hpp`, not sure if these platform-specific headers are intended to be consumed externally (since the whole library is header-only)

Reply:
No, does not work. This was the first I tried.